test/functional/deprecated_rpc.py

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env python3
+# Copyright (c) 2017 The Bitcoin Core developers
+# Distributed under the MIT software license, see the accompanying
+# file COPYING or http://www.opensource.org/licenses/mit-license.php.
+"""Test deprecation of RPC calls."""
+from test_framework.test_framework import BitcoinTestFramework
+from test_framework.util import assert_raises_jsonrpc
+
+class DeprecatedRpcTest(BitcoinTestFramework):
+    def set_test_params(self):
+        self.num_nodes = 2
+        self.setup_clean_chain = True
+        self.extra_args = [[], ["-deprecatedrpc=estimatefee"]]
```

Comment:
I think it would be better to have a dummy RPC that is marked deprecated since `estimatefee` will be removed in the future.

Reply:
hmm, I'm not sure we should add more product code just so the test code can test something else.

We should just change this test to use a different rpc when estimatefee is removed (or skip the test entirely if there are currently no deprecated RPCs).